engines/wintermute/base/font/base_font_bitmap.cpp

```diff
@@ -56,6 +57,9 @@ BaseFontBitmap::BaseFontBitmap(BaseGame *inGame) : BaseFont(inGame) {
 	_fontextFix = false;
 	_freezable = false;
 	_wholeCell = false;
+#ifdef ENABLE_FOXTAIL
```

Comment:
In the interest of save game logic, I guess it might be preferable to unconditionally have this variable, but make it false for all non-foxtail games.

Reply:
See comment below. Maybe _minimizeSpacing  should be removed completely.